ghedesigner/validate.py

```diff
@@ -5,144 +5,36 @@
 
 from jsonschema import ValidationError, validate
 
-from ghedesigner.enums import BHPipeType, DesignGeomType
-
 # Note: JSON schema does not currently have a good way to handle case-insensitive enums.
-#       Some fields are upper-cased manually here for validation purposes.
+#       I think we should enforce case sensitivity.  The validation script should clearly alert the user.
```

Comment:
I am open to suggestion here.  I think it's reasonable to make people play nicely with case sensitivity.  With the validation code now, if you enter a field wrong, here's what you get:

```
 Validation Error:
  Bad Input Location: ground-heat-exchanger → ghe1 → pipe → arrangement
  Problematic Key: arrangement
  Error: 'SingleTube' is not one of ['SINGLEUTUBE', 'DOUBLEUTUBESERIES', 'DOUBLEUTUBEPARALLEL', 'COAXIAL']

 Suggested Fix:  Use one of the allowed values: SINGLEUTUBE, DOUBLEUTUBESERIES, DOUBLEUTUBEPARALLEL, COAXIAL
```

Reply:
Fantastic!